### DISCUSSION TITLE
I have a doubt here.

### DISCUSSION THREAD

**Discussion by itsabinashb** [Jeiwan/uniswapv3-book#65] [Milestone 4. Multi-pool Swaps]
[Here](https://uniswapv3book.com/docs/milestone_4/path/#decoding-first-pool-parameters) we are calling `toUint24()` with only one argument, like this `tickSpacing = path.toUint24(ADDR_SIZE)`, but `toUint24()` takes 2 arguments, `function toUint24(bytes memory _bytes, uint256 _start) internal pure returns(uint24 tempUint){}`, how is it working?

**Comment by Jeiwan** [comment]
This:
https://docs.soliditylang.org/en/v0.8.23/contracts.html#using-for
